parsedmarc/__init__.py

```diff
@@ -48,7 +48,7 @@
 MAGIC_XML = b"\x3c\x3f\x78\x6d\x6c\x20"
 MAGIC_JSON = b"\7b"
 
-IP_ADDRESS_CACHE = ExpiringDict(max_len=10000, max_age_seconds=1800)
+IP_ADDRESS_CACHE = ExpiringDict(max_len=10000, max_age_seconds=14400)
```

Comment:
It may worth making this a config file setting, defaulting to the initial `1800`, as different use-cases may warrant different values.

Reply:
Yes, making it a config file setting may be a good enhancement, but it may take me a while to figure out how to do it.  I can take a look at it after the holiday.

I don't think that there are currently many people locked into the 1,800 setting since it was not working at all until last week.

Reply:
If someone would like to contribute a PR to add that option, be aware that although `IP_ADDRESS_CACHE` is only referenced twice:

https://github.com/domainaware/parsedmarc/blob/master/parsedmarc/__init__.py#L103

https://github.com/domainaware/parsedmarc/blob/master/parsedmarc/__init__.py#L897

configuration items related to report processing must go through several layers of function calls, such as when I just added the new function calls to address the concerns raised in #500.

https://github.com/domainaware/parsedmarc/compare/041296b1f1ba1d0b7b46c2d34f46dd3a2c1ef78c...acef7bdd6ed571f6c049ad6e2f181c0b97bf2e6a